src/triage/cli.py

```diff
@@ -22,6 +42,52 @@ def __call__(self, args):
         print(CONFIG_VERSION)
 
 
+@Triage.register
+class Audition(Command):
+    """ Audition command to run or validate the config file
+    """
+    def __init__(self, parser):
+        parser.add_argument(
+            '-c', '--config',
+            type=argparse.FileType('r'),
+            default='audition_config.yaml',
+            help="config file for audition",
+        )
+        parser.set_defaults(
+            directory=None,
+            validate=True,
+        )
+
+    @cachedproperty
+    def runner(self):
+        db_url = self.root.db_url
+        dir_plot = self.args.directory
+        config = yaml.load(self.args.config)
+        db_engine = create_engine(db_url)
+        return AuditionRunner(config, db_engine, dir_plot)
+
+    def __call__(self, args):
+        self['run'](args)
+
+    @cmdmethod('-d', '--directory', default=None, help="directory to store the result plots from audition")
+    @cmdmethod('-v', '--validate', action='store_true', help="validate before running audition")
+    @cmdmethod('--no-validate', action='store_false', dest='validate', help="run audtion without validation")
+    @cmdmethod('--validate-only', action='store_true', help="only validate the config file not running audition")
+    def run(self, args):
+        if args.validate_only:
+            try:
+                self.runner.validate()
+            except Exception as err:
+                raise(err)
+        elif args.validate:
+            try:
+                self.runner.validate()
+                self.runner.run()
+            except Exception as err:
+                raise(err)
+        else:
+            self.runner.run()
```

Comment:
really not a big deal but still seems like this could be more concise:

    if args.validate or args.validate_only:
        self.runner.validate()

    if not args.validate_only:
        self.runner.run()